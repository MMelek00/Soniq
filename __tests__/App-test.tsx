/**
 * @format
 */

import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import GenericText, {TextType} from '../src/components/genericText';

it('renders correctly', () => {
  const tree = renderer.create(
    <GenericText type={TextType.Error} size={32} text="An error occurred..." />,
  );
  expect(tree).toMatchSnapshot();
});
