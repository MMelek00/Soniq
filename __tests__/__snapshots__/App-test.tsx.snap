// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  size={32}
  style={
    [
      {
        "color": "red",
        "fontSize": 32,
        "fontWeight": "normal",
        "marginBottom": 10,
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 10,
        "textalign": "'center'",
      },
    ]
  }
  type="error"
>
  An error occurred...
</Text>
`;
